docs/site/Authentication-overview.md

```diff
@@ -68,3 +72,7 @@ we've broken down the documentations into several parts:
   Particularly for extension developers.
 - [Use **Express Passport** strategies](Authentication-passport.md): The usage
   of passport adapter module `@loopback/authentication-passport`.
+
+## Examples
+
+There is a list of authentication related examples created by LoopBack team or contributed by community members. They can be found in [authentication examples](https://loopback.io/doc/en/lb4/Examples.html#authentication-and-authorization-related-examples).
```

Comment:
Nitpick: can you use the relative path for the link?

Reply:
Yes, to be consistent with other pages.